encoders/video/VideoPaddleEncoder/requirements.txt

```
@@ -0,0 +1,4 @@
+jina
+paddlepaddle
+paddlehub
+Pillow
```

`visualdl == 2.0.1`, which are required by `paddlepaddle`, has extra requirements,
```suggestion
Pillow
six>=1.14.0
protobuf>=3.11.0
```